src/main/java/com/moodybluez/enterprise/dao/UserSQLDAO.java

```diff
@@ -25,8 +24,7 @@ public User save(User user){
 
     @Override
     public User fetchByUsername(String username){
-        User user = userRepository.findByUsername(username);
-        return user;
+        return userRepository.findByUsername(username);
```

Comment:
Ditto - "No need to define variable and return, just return what the method returns if you don't need to modify it."